Rocket.Chat/Controllers/Chat/ChatControllerUploader.swift

```diff
@@ -110,8 +110,8 @@ extension ChatViewController: UIImagePickerControllerDelegate, UINavigationContr
 
                 file = UploadHelper.file(
                     for: videoData as Data,
-                    name: filename,
-                    mimeType: UploadHelper.mimeTypeFor(videoURL)
+                    name: "\(filename).mp4",
```

Comment:
It looks like it's adding the extension even if filename already have it. Can we check that and only add the `.mp4` extension when it doesn't have it? You can try by simply uploading a video from your iPhone that you recorded your screen (using latest iOS 11). Also, looks like if the extension is `.MOV` it will be `.MOV.mp4`.